sympy/geometry/point.py

```diff
@@ -279,36 +306,43 @@ def is_concyclic(self, *points):
         ========
 
         >>> from sympy import Point
-        >>> from sympy.abc import x
-        >>> p1, p2 = Point(0, 0), Point(1, 1)
-        >>> p3, p4, p5 = Point(2, 2), Point(x, x), Point(1, 2)
-        >>> Point.is_collinear(p1, p2, p3, p4)
+
+        Define 4 points that are on the unit circle:
+
+        >>> p1, p2, p3, p4 = Point(1, 0), (0, 1), (-1, 0), (0, -1)
+
+        >>> p1.is_concyclic() == p1.is_concyclic(p2, p3, p4) == True
         True
-        >>> Point.is_collinear(p1, p2, p3, p5)
+
+        Define a point not on that circle:
+
+        >>> p = Point(1, 1)
+
+        >>> p.is_concyclic(p1, p2, p3)
         False
 
         """
-        args = (self,) + points
-        args = Point._normalize_dimension(*args)
-        args = list(uniq(args))
-        if not Point.affine_rank(*args) <= 2:
+        points = (self,) + args
+        points = Point._normalize_dimension(*[Point(i) for i in points])
```

Comment:
I see that you don't need this since you use the ambient dimension information to raise an error, but I am including it here to make it more transparent and to preserve the identity of the origin, e.g. if the args are `Point(1, 2)` and `Point(1, 2, 3)` the error message references `(-1, -2)` since the error is raised during the shift-to-origin step.